exam_12.js

```diff
@@ -0,0 +1,74 @@
+// ex 1: 
```

Comment:
```suggestion
// Ex 1: 
```
Và ghi rõ đầy đủ từ ra, ko viết tắt